python/graphstorm/config/argument.py

```diff
@@ -747,6 +748,142 @@ def eval_fanout(self):
+        """ filed name for distillation
+        """
+        if hasattr(self, "_concat_field_name"):
+            return self._concat_field_name
+        return None
+
+    @property
+    def tsf_name(self):
+        """ HuggingFace Transformer-based model name for distillation
+        """
+        if hasattr(self, "_tsf_name"):
+            return self._tsf_name
+        return None
+
+    @property
+    def pre_trained_name(self):
+        """ Name of pre-trained HuggingFace model weights for distillation
+        """
+        if hasattr(self, "_pre_trained_name"):
+            return self._pre_trained_name
+        return None
+
+    @property
+    def distill_batch_size(self):
+        """ Batch size for distillation.
+        """
+        if hasattr(self, "_distill_batch_size"):
+            return self._distill_batch_size
+        return None
+
+    @property
+    def need_infer(self):
+        """ Whether to run GNN inference for distillation.
+        """
+        if hasattr(self, "_need_infer"):
+            return self._need_infer
+        return None
+
+    @property
+    def save_student_path(self):
+        """ Saved model path for distillation.
+        """
+        if hasattr(self, "_save_student_path"):
+            return self._save_student_path
+        return None
+
+    @property
+    def distill_lr(self):
+        """ Learning rate for distillation.
+        """
+        if hasattr(self, "_distill_lr"):
+            return self._distill_lr
+        return None
+
+    @property
+    def save_student_interval(self):
+        """ Model checkpoint interval for distillation.
+        """
+        if hasattr(self, "_save_student_interval"):
+            return self._save_student_interval
+        return None
+
+    @property
+    def eval_student_interval(self):
+        """ Validation interval for distillation.
+        """
+        if hasattr(self, "_eval_student_interval"):
+            return self._eval_student_interval
+        return None
+
+    @property
+    def chunk_size(self):
+        """ Size of each file for distillation data.
+        """
+        if hasattr(self, "_chunk_size"):
+            return self._chunk_size
+        return None
+
+    def _check_max_global_step(self, max_global_step):
+        """ Parse maximum global size for distillation for each node type.
+        """
+        max_global_step_dict = {}
+        try:
+            for node_step in max_global_step.split(","):
+                node_step_split = node_step.split(":")
+                max_global_step_dict[node_step_split[0]] = int(node_step_split[1])
+
+        except Exception:
+            assert False, f"{fot_name} max_global_step should in format" \
+                "ntype1:step1,ntype2:step2, ..." \
+                "for example, a valid input can be 'query:512@asin:512'"
+        return max_global_step_dict
+
+    @property
+    def max_global_step(self):
+        """ Maximum global size for distillation.
```

Comment:
what is global size?

Reply:
user can specify the number of global steps for distillation training. Otherwise the training will go infinitely.